spec/factories/ingredients.rb

```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
   factory :ingredient do
-    name { Faker::Food.ingredient }
+    name { Faker::Food.unique.ingredient }
```

Comment:
This is going to make life so much easier. That was an annoying thing to fail a test on. Thanks!